### ISSUE TITLE
[3.3.5][6.x][Shaman]Tier 6 2P

### ISSUE THREAD

**Issue by w5860363** [TrinityCore/TrinityCore#13930]
core:https://github.com/TrinityCore/TrinityCore/commit/603f49a55134de21b6c02b3cfcd9cb32b44c5f7b
DB:TDB59
update:2015_08_23_00_world.sql
src/server/game/Spells/Auras/SpellAuraEffects.cpp

case 38443:
// Totemic Mastery (Skyshatter Regalia (Shaman Tier 6) - bonus)
                    
Don't work
http://www.wowhead.com/itemset=684
http://www.wowhead.com/spell=38443/totemic-mastery

**Comment by Aokromes** [comment]
https://github.com/TrinityCore/TrinityCore/blob/6.x/CONTRIBUTING.md

**Comment by Keader** [comment]
@Aokromes Could reopen?
Confirm  rev b28849003c881d0a05624195d435ea903c13d4f8
Effect : Increases Spirit by 38, Critical Strike by 35, and Intellect by 45. dont work

**Comment by Keader** [comment]
```diff

diff --git a/src/server/game/Spells/Auras/SpellAuraEffects.cpp b/src/server/game/Spells/Auras/SpellAuraEffects.cpp
index f749117..22f7a48 100644
--- a/src/server/game/Spells/Auras/SpellAuraEffects.cpp
+++ b/src/server/game/Spells/Auras/SpellAuraEffects.cpp
@@ -5646,25 +5646,6 @@ void AuraEffect::HandlePeriodicTriggerSpellAuraTick(Unit* target, Unit* caster)
                             target->RemoveAurasDueToSpell(28820);
                         return;
                     }
-                    // Totemic Mastery (Skyshatter Regalia (Shaman Tier 6) - bonus)
-                    case 38443:
-                    {
-                        bool all = true;
-                        for (int i = SUMMON_SLOT_TOTEM; i < MAX_TOTEM_SLOT; ++i)
-                        {
-                            if (!target->m_SummonSlot[i])
-                            {
-                                all = false;
-                                break;
-                            }
-                        }
-
-                        if (all)
-                            target->CastSpell(target, 38437, true, NULL, this);
-                        else
-                            target->RemoveAurasDueToSpell(38437);
-                        return;
-                    }
                 }
                 break;
             }
@@ -5755,6 +5736,26 @@ void AuraEffect::HandlePeriodicTriggerSpellAuraTick(Unit* target, Unit* caster)
                     caster->CastSpell(target, triggerSpellId, false);
                 return;
             }
+			// Totemic Mastery (Skyshatter Regalia (Shaman Tier 6) - bonus)
+			case 38443:
+			{
+				bool all = true;
+				for (int i = SUMMON_SLOT_TOTEM; i < MAX_TOTEM_SLOT; ++i)
+				{
+					if (!target->m_SummonSlot[i])
+					{
+						all = false;
+						break;
+					}
+				}
+
+				if (all)
+					target->CastSpell(target, 38437, true, NULL, this);
+				else
+					target->RemoveAurasDueToSpell(38437);
+				return;
+			}
+			break;
         }
     }
 
```

I do not know if it is considered "hack", but it works

**Comment by Rushor** [comment]
@Keader : you know how to pr ^^

**Comment by Keader** [comment]
Closed by: 9b796ddf0b82f81425a5b843695c47d1d9a47e6e
@Rushor  close this please xD
